tests/backup/backup_share_test.go

```diff
@@ -3450,7 +3453,7 @@ var _ = Describe("{IssueMultipleRestoresWithNamespaceAndStorageClassMapping}", f
 			err = ShareBackup(backupName, nil, userName, FullAccess, ctx)
 			dash.VerifyFatal(err, nil, fmt.Sprintf("Share backup %s with  user %s having FullAccess", backupName, userName))
 			userBackups1, _ := GetAllBackupsForUser(userName[0], commonPassword)
```

Comment:
If it is not much, how about we handle the error here?

Reply:
This PR is mainly for StorkUpgradeWithBackup, will handle all ther test modification in another PR.